DependencyInjection/AbstractSonataAdminExtension.php

```diff
@@ -43,7 +43,7 @@ protected function fixTemplatesConfiguration(array $configs, ContainerBuilder $c
                         'datetime' => 'SonataAdminBundle:CRUD:list_datetime.html.twig',
                         'text' => 'SonataAdminBundle:CRUD:list_string.html.twig',
                         'textarea' => 'SonataAdminBundle:CRUD:list_string.html.twig',
-                        'email' => 'SonataAdminBundle:CRUD:list_string.html.twig',
+                        'email' => 'SonataAdminBundle:CRUD:list_email.html.twig',
```

Comment:
@Soullivaneuh this is not hard, but it is a BC break....

IMO we can merge it anyway in `3.x`

What do you say @greg0ire @core23 ?